### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getPath' for undefined

### ISSUE THREAD

**Issue by Centaur** [atom-haskell/haskell-ghc-mod#37]
Affect version 0.8.6 . This may be related to #31 

Reproduce: 
1. Main Menu -> View -> Toggle Tree View make sure the left treeview is visible 
2. remove any project folder in the left treeview
3. Main Menu -> Atom -> Preferences , now in the left treeview there will be an empty node named 'config'
4. open any `.hs` file
5. move mouse over a method or a variable , e.g. `putStrLn`, haskell-ghc-mod will show its type
6. right click on 'config' node in the left treeview and select 'remove project folder'
7. repeat 5 will cause an error 'Uncaught TypeError: Cannot read property 'getPath' of undefined' 

Here is the stacktrace: 

```
/Users/user/.atom/packages/haskell-ghc-mod/lib/util.coffee:16
Hide Stack Trace
TypeError: Cannot read property 'getPath' of undefined
    at Object.module.exports.Util.getRootDir (/Users/user/.atom/packages/haskell-ghc-mod/lib/util.coffee:28:14)
    at GhcModiProcess.module.exports.GhcModiProcess.getTypeInBuffer (/Users/user/.atom/packages/haskell-ghc-mod/lib/ghc-modi-process.coffee:233:19)
    at GhcModiProcess.getTypeInBuffer (/Users/user/.atom/packages/haskell-ghc-mod/lib/ghc-modi-process.coffee:3:61)
    at IdeBackend.module.exports.IdeBackend.getType (/Users/user/.atom/packages/haskell-ghc-mod/lib/ide-backend.coffee:81:29)
    at IdeBackend.getType (/Users/user/.atom/packages/haskell-ghc-mod/lib/ide-backend.coffee:3:61)
    at /Users/user/.atom/packages/ide-haskell/lib/editor-control.coffee:272:97
    at EditorControl.showExpressionType (/Users/user/.atom/packages/ide-haskell/lib/editor-control.coffee:329:16)
    at /Users/user/.atom/packages/ide-haskell/lib/editor-control.coffee:79:57
```

**Comment by lierdakil** [comment]
Sorry, my bad, messed up fix for #31. Sleep deprivation does that. 0.8.7 should be fine.

**Comment by Centaur** [comment]
Confirm fixed in 0.8.7 (restart Atom after upgrade).
